api/openapi.yaml

```diff
@@ -207,6 +207,38 @@ paths:
                 type: array
                 items:
                   type: string
+  /tests/{testId}/run/{runId}/dry-run:
+    put:
+      tags:
+        - api
+      parameters:
+        - in: path
+          name: testId
+          schema:
+            type: string
+            format: uuid
+          required: true
+        - in: path
+          name: runId
+          schema:
+            type: string
+            format: uuid
+          required: true
+      summary: "run given assertions agains the traces from the given run without persisting anything"
+      description: "use this method to test a definition agains an actual trace without creating a new version or persisting anything"
+      operationId: assertionDryRun
```

Comment:
That doesn't seem like a method name. What do you think about renaming that to `dryRunAssertion`?